source/braille.py

```diff
@@ -878,10 +886,15 @@ def _addFieldText(self, text, contentPos):
 		self.rawTextTypeforms.extend((louis.plain_text,) * textLen)
 		self._rawToContentPos.extend((contentPos,) * textLen)
 
-	def _addTextWithFields(self, info, formatConfig, isSelection=False):
+	def _addTextWithFields(self, info, formatConfig, isSelection=False, useCache=True):
 		shouldMoveCursorToFirstContent = not isSelection and self.cursorPos is not None
 		ctrlFields = []
 		typeform = louis.plain_text
+		if useCache:

```

Comment:
Using an empty dictionary is semantically a bit different from having no cache I guess. But if we find that acceptable, then we don't even have to check for `useCache` here.
However, note that I stole this code from speech.py, and I think it's good to keep those similar in code style.